crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/rpc.rs

```diff
@@ -39,7 +39,13 @@ pub enum CallResult {
     Failure(CallFailure),
 }
 
-/// Enum representing possible call failure and its' type.
+impl Default for CallResult {
+    fn default() -> Self {
+        Self::Failure(CallFailure::Error { msg: String::new() })
+    }
+}
```

Comment:
why?

Reply:
To avoid boilerplate code when creating `CallTrace`, I can write `..Default::default()` now

Reply:
Imo this hides some important information.

Reply:
You would literally never leave it default though, I can revert this if you want to

Reply:
discussed offline `build_failed` function should be used instead.

Reply:
Used `default_successful_call` instead